x-pack/plugins/data_enhanced/public/search/ui/background_session_indicator/background_session_indicator.scss

```diff
@@ -0,0 +1,3 @@
+.backgroundSessionIndicator__secondaryActionContainer {
+  border-left: 1px solid $euiColorLightShade;
```

Comment:
@mdefazio should check the overall design, but one quick observation is that you can write this as:

```
border-left: $euiBorderWidthThin;
```